### PR TITLE
Docker files to be able to launch the selenium tests with docker-compose

### PR DESCRIPTION
**Pull request by Quetzacoalt91**

### CODE REVIEW

File: wait-for-it.sh [hunk 1]
```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
+
```
Comment (Quetzacoalt91) [inline wait-for-it.sh]: I guess this one should be deleted, and we keep `docker_prestashop/wait-for-it.sh` instead?

File: runTests.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+PS_VERSION="${PS_VERSION:-1.7}"
+URL="${URL:-prestashop}"
+MODULE="${MODULE:-}"
+SAUCELABS="${SAUCELABS:-}"
+SCRIPT="${SCRIPT:-index.webdriverio.js}"
+
+COMMAND="/tmp/node_modules/mocha/bin/mocha /tmp/test/itg/$PS_VERSION/$SCRIPT --URL=$URL"
+
+if [ -n "$MODULE" ]; then
+    COMMAND="$COMMAND --MODULE=$MODULE"
```
Comment (Quetzacoalt91) [inline runTests.sh:13]: To be checked with @basma-yangui-prestashop. If I remember, we always need to set a value here, `none` if we do not test any module.

We may need to update the JS code as well?

File: runTests.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+fi
+
+if [ -n "$SAUCELABS" ]; then
+    COMMAND="$COMMAND --SAUCELABS"
```
Comment (Quetzacoalt91) [inline runTests.sh:17]: Same comment here. @basma-yangui-prestashop can you confirm removing the parameter `SAUCELABS` will work?

**Review by Quetzacoalt91 — COMMENTED** [review]
First review prior test deployment :)

File: runTests.sh (hunk 2 above)
Comment (basma-yangui-prestashop) [reply runTests.sh:13]: It's OK @Quetzacoalt91 . If we do not test any module, the MODULE option can be omitted.

File: runTests.sh (hunk 3 above)
Comment (basma-yangui-prestashop) [reply runTests.sh:17]: Like the Module option, the SAUCELABS option can be removed if it is not needed.

**Comment by Quetzacoalt91** [comment]
The tests are broken on PrestaShop 1.7, but this failure is not related to this PR.

**Comment by Quetzacoalt91** [comment]
The selenium server is not available on localhost:4444, we need to make the value updatable.

```
tests_1            | 
tests_1            |   0 passing (89ms)
tests_1            |   2 failing
tests_1            | 
tests_1            |   1) Allscenario "before all" hook:
tests_1            |      Uncaught connect ECONNREFUSED 127.0.0.1:4444
tests_1            |   Error: Couldn't connect to selenium server
tests_1            |       at saveScreenshot("/tmp/test/itg/1.7/screenshots/chrome_exception_1497450053542_undefined.png") - common.webdriverio.js:114:16
tests_1            | 
tests_1            |   2) Allscenario "after all" hook:
tests_1            |      Uncaught connect ECONNREFUSED 127.0.0.1:4444
tests_1            |   Error: Couldn't connect to selenium server
tests_1            |       at saveScreenshot("/tmp/test/itg/1.7/screenshots/chrome_exception_1497450053542_undefined.png") - common.webdriverio.js:114:16
tests_1            | 
tests_1            | 
tests_1            | 
psfunctionaltests_tests_1 exited with code 2
```

**Comment by Quetzacoalt91** [comment]
Current issue with this, the module `welcome` is removed too early.

```
tests_1            |   26 passing (19m)
tests_1            |   29 failing
tests_1            | 
tests_1            |   1) Allscenario The Product Creation Create new product should click on the <add new product> button:
tests_1            |      Uncaught unknown error: Element <li class="maintab  has_submenu" id="subtab-AdminCatalog" data-submenu="9">...</li> is not clickable at point (105, 165). Other element would receive the click: <div class="onboarding-popup bootstrap">...</div>
tests_1            |   Error: An unknown server-side error occurred while processing the command.
tests_1            |       at elementIdClick("4") - click.js:20:22
```

**Pull request merged**